Sources/MongoSwift/MongoDatabase.swift

```diff
@@ -1,17 +1,17 @@
 import mongoc
 
-/// Options to set on a retrieved `MongoCollection`.
+/// Options to set on a retrieved `MongoCollection`. If an option is not specified here, the collection will inherit the
```

Comment:
similar to my comment above, this might make more sense if it were in the docstring for `MongoDatabase.collection` (both the one that gives you back a `MongoCollection<Document>` as well as the `withType` variant.)